foundations/likvid-prod/platforms/azure/buildingblocks/starterkit/terragrunt.hcl

```diff
@@ -11,8 +11,8 @@ include "platform" {
 # todo: not quite sure how this will interact with our ability to generate docs in CI/CD for this module
 # it _should_ be fine since we only need to read terraform statex^^
 
-dependency "sandbox" {
```

Comment:
f: please roll back this change as discussed, we currently designed and documented the starterkit to be only eligible for sandbox projects (ie experimentation, non-prod)

the current reasoning here is that going productive with a use case should be a fork + own of the code, with the CFT no longer owning key automation infra (like the azure managed identity) but instead having the application team taking full responsibility